builtin/providers/aws/resource_aws_elastic_beanstalk_environment.go

```diff
@@ -249,7 +249,7 @@ func resourceAwsElasticBeanstalkEnvironmentCreate(d *schema.ResourceData, meta i
 		Refresh:    environmentStateRefreshFunc(conn, d.Id()),
 		Timeout:    waitForReadyTimeOut,
 		Delay:      10 * time.Second,
```

Comment:
Delay is the amount of time before we start. MinTimeout is the minimum amount of time between refreshes:

```go
Delay          time.Duration    // Wait this time before starting checks
MinTimeout     time.Duration    // Smallest time to wait before refreshes
```

I'll be refactoring this PR to expose at least `MinTimeout`, stay tuned!